package.json

```diff
@@ -23,25 +23,26 @@
     "gulp": "*"
   },
   "dependencies": {
+    "babel-eslint": "^7.1.1",
     "del": "^2.0.2",
-    "godaddy-style": "^3.0.0",
-    "gulp-eslint": "^2.0.0",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-mocha": "^4.8.0",
+    "eslint-plugin-react": "^6.10.0",
+    "godaddy-style": "^4.0.1",
```

Comment:
Some packages that use godaddy-test-tools may not have been updated for ES6, so they need access to the es5 rules. It's possible for individual packages to use whatever method they choose, so supporting everything here makes sense for this tool.